### ISSUE TITLE
Virustotal reporting matching rules causing downranking

### ISSUE THREAD

**Issue by silversword411** [SigmaHQ/sigma#2183]
Trying to follow this: https://www.virustotal.com/gui/file/521bc8860c8c1eab034777e8e5bd9dadbdacf2b24a7c475f284cb8d50a3cc20b/detection

The
Stop Windows Service by Jakob Weinzettl seems to report on our agent, why is this a concern and listed in the problem report?

**Comment by wagga40** [comment]
> I saw that...but I'm trying to find what's getting the global thumbs down for many AV's since all the matching rules are low/informational only.

As you certainly know, the AV detections are related to the binary itself rather than the results of the sigma rules.

**Comment by silversword411** [comment]
> As you certainly know, the AV detections are related to the binary itself rather than the results of the sigma rules.

Maybe back in the day, but that's impractical now. Just change a character in source, and recompile...also known as "heuristic viruses".

Also all signed TRMM installers (like this one) are actually custom built, and signed with a developer key. Also they have customized command parameters built in which also differentiates every TRMM installer. 

I believe the global thumbs up/down rating is like spam rating. Many things to into generate a total "score" and once it passes a certain threshold it goes from one to another. I think those rules (especially in-aggregate) have an effect on the final outcome. 

If you have more detailed descriptions from AV vendors that describe things further I'd appreciate any links for further knowledge.

**Comment by Neo23x0** [comment]
> I think those rules (especially in-aggregate) have an effect on the final outcome.

That's not the case. 

As said before: this is a rule with level "low", which means, that it has an informational character. 
AV vendors don't write detections for samples that have a matching Sigma rule with level low. The don't do it even for level critical. They would first look at a sample and then decide if it's really malicious before they write a rule based on a community vote and remove some legitimate software from customers' systems.

**Comment by Neo23x0** [comment]
if I were you, I would rather change the contents of my software to look less like malware. 

E.g. I'd change that name "downloadAgent" to something else. 

I see many, many strings and functions that looks suspicious and very similar to Go-based RATs. 
https://github.com/wh1te909/rmmagent/tree/master/agent

You tools seems to be a legitimate RAT. 
And don't try to obfuscate these strings by encoding them with base64. That would trigger other rules.

**Comment by silversword411** [comment]
> if I were you, I would rather change the contents of my software to look less like malware.

Uh...don't build what you're building...go make flower arrangements?!?!?

> E.g. I'd change that name "downloadAgent" to something else.

So don't base64 obfuscate, but Function name obfuscate? Again, that makes absolutely no sense. 

> You tools seems to be a legitimate RAT.

It's RMM software (Remote Management and Monitoring software....basically a legal RAT), just like Connectwise Automate, Kaseya, solarwinds...sorry N-Able, Atera etc. etc. etc
Flag them as malware too, and then we'll all be on the same playing field. Right now Tactical RMM is being treated differently than closed-source RMM software that does exactly the same functionality which is what I'm trying to fix/find out the official process on.

--------------------------

This Github issue is about trying to figure out how to train/communicate with 50+ AV. I got the impression the low rules were one small part of why AV was flagging it as Malware

There's 2 versions of the agent: Unsigned (and open source). Can malicious actors use it for bad purposes? Of course. So flag the unsigned as PUA.

There's another that's code signed for 6+ months, and requires a subscription/payment and is trackable thru the code-signing developer. If it's used maliciously, follow the trail and jail the FU#*$@!.

Any useful information along dealing with AV vendors would be appreciated. Methods and best-practices. Communication methods etc.

**Comment by frack113** [comment]
I start reading the virustotal [documentaion](https://support.virustotal.com/hc/en-us/articles/360015738658-Sigma-rules)
The example have a green note of 0 but in the same time sigma 1 critical / 1 high / 1 medium / 2 low.

I don't understand the connection you make between the antivirus and the sigma rules.
It is Antivirus issue not realy a sigma.
Did you try on [joesandbox](https://www.joesandbox.com/#windows) to see the result as there is sigma rules too there ?

Thanks

**Comment by silversword411** [comment]
> I don't understand the connection you make between the antivirus and the sigma rules.

Maybe I misunderstand sigma rules, but I would assume it's a factor in how AV evaluate software to classify it:

- malware
- PUA (Potentially Unwanted Applications)
- Safe

etc

Since Virustotal is listing it, it would see important.

Thanks

**Comment by frack113** [comment]
Even if I did a malware reverse engineering training and some rootme chalange, I am not a specialist.

- Sigma rules are made to detect artifacts left on the system in the log.
- Antivirus detect the behaviour of the execution OR signature

In the rapport, you got 38/69 **antivirus** detection.
Some of your code make the same behaviour than malware.
Make a false positive request to [virustotal](https://www.virustotal.com/gui/contact-us )

In conclusion Sigma is not a antivirus.
Malware or hacker leave artifact to us make sigma rules 😄 

Thanks

**Comment by Neo23x0** [comment]
> Maybe I misunderstand sigma rules

Yes, you do.

> but I would assume it's a factor in how AV evaluate software to classify it

No.
